Maths/test/AverageMedian.test.js

```diff
@@ -18,4 +18,4 @@ test('should return the median of an array of numbers:', () => {
 test('should return the median of an array of numbers:', () => {
   const medianValue = averageMedian([1, 2, 3, 4, 6, 8])
   expect(medianValue).toBe(3.5)
-})
+})
```

Comment:
Dont remove ending newline.

Reply:
yeah, thx, i forget about ur CONTRIBUTING.md

one indication would be enough)

Reply:
total: 10 comments and 4 commits for two lines of code)))

Reply:
Those two lines of code were the crux of what you were trying to achieve. All the other code was to try and generalize this method, which is not required in this context. But even small contributions are useful in making this repository better.